ios/gamma/split-view/RNSSplitViewScreenController.swift

```diff
@@ -69,16 +77,77 @@ public class RNSSplitViewScreenController: UIViewController {
 
   // MARK: Layout
 
+  ///
+  /// @brief This method is overridden to extract the value to which we're transitioning
+  /// and attach the DisplayLink to track frame updates on the presentation layer.
+  ///
+  public override func viewWillTransition(
+    to size: CGSize,
+    with coordinator: any UIViewControllerTransitionCoordinator
```

Comment:
Btw, what about this coordinator? Doesn't he have the same information as I mentioned when we talked about screen transitions? Seems like it to me

Reply:
```swift
coordinator.animate(alongsideTransition: { _ in
          DispatchQueue.main.async {
              print("TEST")
          }
 })
```

for this snippet, I was receiving 1 log always (assuming that `alongsideTransition` should give me more logs itself)

Reply:
It doesn't work that way here, I think you should rather mimic something close to it: https://github.com/software-mansion/react-native-screens/blob/d3aef2558d19987dedbac70e77a6bbeed91139fa/ios/RNSScreen.mm#L1732-L1741. So still have the displayLink but with correct completion event.

Reply:
https://github.com/software-mansion/react-native-screens/pull/3073/commits/1cd9f34753845c541b85865db922f196eab68009